app/e2e/common/pages/playwright-landing-page.ts

```diff
@@ -0,0 +1,158 @@
+// playwright-landing-page.ts
+import { expect, Locator, Page } from '@playwright/test';
+
+interface RegisterForm {
+    email: string;
+    password: string;
+    firstName: string;
+    lastName: string;
+}
+
+interface LoginForm {
+    email: string;
+    password: string;
+}
```

Comment:
We could potentially use type generation here. Re-typing every single form would be painful.

Reply:
I'd like to discuss this idea in more detail next time we meet.